docs/BigQueryMultiTable-batchsink.md

```diff
@@ -1,66 +1,77 @@
-* When this is set to false, any mismatches between the schema expected by the pipeline 
-and the schema in BigQuery will result in pipeline failure. 
-* When this is set to true, the schema in BigQuery will be updated to match the schema 
-expected by the pipeline, assuming the schemas are compatible. 
-
-Compatible changes fall under the following categories:                
-* the pipeline schema contains nullable fields that do not exist in the BigQuery schema. 
-In this case, the new fields will be added to the BigQuery schema.
-* the pipeline schema contains nullable fields that are non-nullable in the BigQuery schema. 
-In this case, the fields will be modified to become nullable in the BigQuery schema.
-* the pipeline schema does not contain fields that exist in the BigQuery schema. 
-In this case, those fields in the BigQuery schema will be modified to become nullable.
-                         
-Incompatible schema changes will result in pipeline failure.
-
-**Service Account File Path:** Path on the local file system of the service account key used for
-authorization. Can be set to 'auto-detect' when running on a Dataproc cluster.
-When running on other clusters, the file must be present on every node in the cluster.
-
-Example
--------
+* **Split Field**
+
+  The name of the field that will be used to determine which table to write to.
+Defaults to `tablename`.
+
+* **Update Table Schema**
+
+  Whether the BigQuery table schema should be modified when it does not match the schema expected by the pipeline.
+   * When this is set to false, any mismatches between the schema expected by the pipeline and the schema in
+   BigQuery will result in pipeline failure.
+   * When this is set to true, the schema in BigQuery will be updated to match the schema expected by the
+   pipeline, assuming the schemas are compatible.
+
+  Compatible changes fall under the following categories:
+
+    * the pipeline schema contains nullable fields that do not exist in the BigQuery schema. In this case,
+    the new fields will be added to the BigQuery schema.
+    * the pipeline schema contains nullable fields that are non-nullable in the BigQuery schema. In this case,
+    the fields will be modified to become nullable in the BigQuery schema.
+    * the pipeline schema does not contain fields that exist in the BigQuery schema.
+
+  In this case, those fields in the BigQuery schema will be modified to become nullable. Incompatible schema
+  changes will result in pipeline failure.
+
+#### **Project and Credentials**
+
+If the plugin is run in GCP environment, the service account file path does not need to be
+specified and can be set to 'auto-detect'. Credentials will be automatically read from the GCP environment.
+A path to a service account key must be provided when not running in GCP. The service account
+key can be found on the Dashboard in the Cloud Platform Console. Ensure that the account key has permission
+to access resource.
+
+* **Project Id**
```

Comment:
Why not ? I have seen customers when they confused when it's along with other configuration. Even though they are properties that need to be filled grouping them is easier and in future we will turn them into tabs.

Reply:
Because the project is used to identify *what* you are reading from and not *who* is reading. It's not related to credentials or auth at all. I can use a service account from a different project. Why should they be together?